chat.js

```diff
@@ -916,8 +916,13 @@ Chat.loadCommands = function () {
 	Object.assign(commands, require('./chat-plugins/info').commands);
 
 	for (let file of fs.readdirSync(path.resolve(__dirname, 'chat-plugins'))) {
-		if (file.substr(-3) !== '.js' || file === 'info.js') continue;
+		if (file.substr(-3) !== '.js' || file === 'info.js' || file === 'github.js') continue;
 		Object.assign(commands, require('./chat-plugins/' + file).commands);
+	}	
+
+	try {
```

Comment:
This will ignore syntax errors and other issues like that, sounds like a bad idea.